### PR TITLE
new page for Insights

### PR DESCRIPTION
**Pull request by michelle-luna**

### CODE REVIEW

File: jekyll/_cci2/insights.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+*[Basics]({{ site.baseurl }}/2.0/basics/) > Using Insights*
+
+This document describes creating and using insights in CircleCI in the following sections:
+
+* TOC
+{:toc}
+
+## Overview
+
+CircleCI Insights shows you how your tests are performing with integrated graphs and a dashboard for performance of all of your code repositories.
```
Comment (KunalJain) [inline jekyll/_cci2/insights.md]: Maybe something like this:
Insights dashboard provides you a snapshot the health of you repos that you are following on CircleCI. At glance you can see median build, median queue, last build, success rate and parallelism for your default branch. Note: If you have configured workflows, graphs are going to show you all the jobs that are getting executed for your default branch.

File: jekyll/_cci2/insights.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Project Insights
+
+Click the Insights icon on the main navigation, then click your repo name to access per-project insights.
+
+The per-project insights page gives you access to the build status and build performance graphs for a selected branch.
+
+![header]({{ site.baseurl }}/assets/img/docs/insights-current-build.png)
+
+- **Build Status:** The Insights dashboard shows the last 50 builds for your default branch. Click a branch to access over 100 build statuses for the selected branch.
```
Comment (KunalJain) [inline jekyll/_cci2/insights.md]: Click a branch on the top right corner to access over 100 build/job statuses for the selected branch.

File: jekyll/_cci2/insights.md (hunk 1 above)
Comment (michelle-luna) [reply jekyll/_cci2/insights.md]: thanks, I changed to "Click the Insights menu item in the CircleCI app to view a dashboard showing the health of all repositories you are following. Median build time, median queue time, last build time, success rate, and parallelism appear for your default branch. **Note:** If you have configured Workflows, graphs display all of the jobs that are being executed for your default branch."

File: jekyll/_cci2/insights.md (hunk 2 above)
Comment (michelle-luna) [reply jekyll/_cci2/insights.md]: thanks, added

File: jekyll/_cci2/insights.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Click the Insights icon on the main navigation, then click your repo name to access per-project insights.
+
+The per-project insights page gives you access to the build status and build performance graphs for a selected branch.
+
+![header]({{ site.baseurl }}/assets/img/docs/insights-current-build.png)
+
+- **Build Status:** The Insights dashboard shows the last 50 builds for your default branch. Click a branch in the top right corner to access over 100 build/job statuses for the selected branch.
+
+- **Build Performance:** The Build Performance graph aggregates your build data for a particular day and plots the median for that day going back as far as 90 days. Monitor the performance of your repo by clicking a particular branch.
```
Comment (KunalJain) [inline jekyll/_cci2/insights.md]: We should do the same here: `build/job data'

**Comment by KunalJain** [comment]
Added one more comment other than that looks good to me :)

**Pull request merged**